test/knexfile.js

```diff
@@ -30,10 +30,14 @@ module.exports = [
         }
     }, {
         client: 'postgres',
+        pool: {
```

Comment:
What do these settings do?

Reply:
After each test I destroy the knex for the TestSession, at the beginning of each test I create a new TestSession. This is because I needed to modify knexes for certain error cases.

There seems to be some overlap of resources or these don't get shutdown perfectly in-between tests. Without these pool settings, I kept getting an error from knex saying `too many connections` -- when searching onl

Reply:
We should ensure `knex.destroy()` is reliably getting called.  That likely means using lab's `onCleanup`.  If `destroy()` doesn't work properly, it would be good for us to know and potentially file as a bug or fix!